components/nameplate/NameplateGroup.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Nameplate, {NameplateProps, Overflow} from './Nameplate';
+
+export interface NameplateGroupProps {
+  names: NameplateProps[];
+  expandCurrentUser?: boolean;
+  limit?: number;
+}
+
+function NameplateGroup({
+  names,
+  expandCurrentUser = false,
+  limit,
+}: NameplateGroupProps): JSX.Element {
+  limit = limit ?? names.length;
```

Comment:
im surprised eslint is okay with this.. i thought parameters were considered `const`